mucommander-viewer-text/src/main/java/com/mucommander/viewer/text/TextEditor.java

```diff
@@ -133,6 +133,25 @@ protected void initMenuBarItems() { // TODO code dup with TextViewer, fix it
 
                 viewMenu = new JMenu(Translator.get("text_editor.view"));
 
+                JMenu tabSyntaxMenu = new JMenu(Translator.get("text_editor.syntax_style"));
+
+                ButtonGroup group = new ButtonGroup();
+                for (String syntaxStyle : textEditorImpl.getSyntaxStyles()) {
+                    // TODO reflect the current style (that was automagically set on open)
+                    JRadioButtonMenuItem radio = new JRadioButtonMenuItem(syntaxStyle, false);
+                    radio.addActionListener(
+                            e -> {
+                                boolean saveNeededKeeper = isSaveNeeded();
+                                textEditorImpl.setSyntaxStyle(syntaxStyle);
+                                setSaveNeeded(saveNeededKeeper);
+                            }
+                    );
```

Comment:
same comment as below on TextViewer

Reply:
not sure if it is relevant, or I don't get it :)

Reply:
I'm not sure now what I meant by that either :)